app/components/provider-links/component.js

```diff
@@ -0,0 +1,30 @@
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+
+export default Component.extend({
+    store: service(),
+    theme: service(),
+
+    tagName: 'ul',
+    classNames: ['fa-ul'],
+
+    init() {
+        this._super(...arguments);
+        this.iconMap = {
+            preprint: 'fa-graduation-cap',
+            paper: 'fa-file-text-o',
+            thesis: 'fa-book',
+        };
+        this.pendingCount = null;
+        this.get('fetchData').perform();
+    },
+
+    fetchData: task(function* () {
+        const provider = yield this.get('theme').loadProvider(this.get('provider.id'));
```

Comment:
It feels wrong to use `theme` on this non-provider-specific view. Could just ask `store` for it.

Reply:
Yeah I tried using `store` first but it didn't work. I'll try again though because it might have been user error.